yogstation/code/modules/spells/cauterize.dm

```diff
@@ -0,0 +1,35 @@
+/obj/effect/proc_holder/spell/targeted/cauterize
+	name = "Cauterize"
+	desc = "Replace all brute/burn damage with burn damage, taken over time. Higher levels increases time taken to return back to original damage level."
+	clothes_req = TRUE
+	human_req = FALSE
+	charge_max = 800
+	cooldown_min = 600 //50 deciseconds reduction per level
+	range = -1
+	var/cauterize_duration = 20 //in seconds
+	include_user = TRUE
+	invocation = "HEA'OW!" //someone come up with a good invocation i beg you
```

Comment:
Maybe it could be the fireball invocation in reverse?

Reply:
Good idea. Done.